test/dhcp/record_test.rb

```diff
@@ -56,7 +47,16 @@ def test_should_not_save_invalid_subnets
     end
   end
 
-  def test_options_should_be_a_hash
-    assert_kind_of Hash, @record.options
```

Comment:
By default the constructor argument is a hash, otherwise options will be initialized with w/e arguments were passed to the constructor. All this test does is verifies that setup code passes in hash as a parameter.